website/sidebar-content-parser.ts

```
@@ -0,0 +1,95 @@
+
+/**
+ * Generate JSON overview file for navigation
+ * @param sidebarItems - Sidebar items from Docusaurus
+ * @param section - Section type ('docs' or 'tutorial')
+ * @param baseUrl - Base URL for the site
+ * @param outputPath - Path to output the JSON file
+ */
+export async function generateJsonOverviewFile(
+  sidebarItems: SidebarItem[],
+  section: 'docs' | 'tutorial',
+  baseUrl: string = 'https://podman-desktop.io',
```

Biome (2.1.2)

[error] This default parameter should follow the last required parameter or should be a required parameter.

The last required parameter is here:

A default parameter that precedes a required parameter cannot be omitted at call site.
Unsafe fix: Turn the parameter into a required parameter.


(lint/style/useDefaultParameterLast)